astronomer/providers/snowflake/triggers/snowflake_trigger.py

```diff
@@ -62,7 +62,7 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
         """
         hook = get_db_hook(self.snowflake_conn_id)
         try:
-            run_state = await hook.get_query_status(self.query_ids)
+            run_state = await hook.get_query_status(self.query_ids, self.polling_period_seconds)
```

Comment:
I think we should keep variable names consistent in operator/sensor/hook/trigger. So please consider renaming `polling_period_seconds` to `polling_inverval` since the operator has `polling_inverval` and at most of the places for polling we have `polling_inverval` variable name

Reply:
Changed it to `poll_interval`